cmake/modules/RootBuildOptions.cmake

```diff
@@ -65,7 +65,7 @@ ROOT_BUILD_OPTION(afs OFF "AFS support, requires AFS libs and objects")
 ROOT_BUILD_OPTION(alien OFF "AliEn support, requires libgapiUI from ALICE")
 ROOT_BUILD_OPTION(asimage ON "Image processing support, requires libAfterImage")
 ROOT_BUILD_OPTION(astiff ON "Include tiff support in image processing")
-ROOT_BUILD_OPTION(bonjour ON "Bonjour support, requires libdns_sd and/or Avahi")
+ROOT_BUILD_OPTION(bonjour OFF "Bonjour support, requires libdns_sd and/or Avahi")
```

Comment:
This option does not work for me, I'm not sure it's even worth keeping around if the code is broken and unmaintained.

Reply:
Do you mean bonjour OFF or ON?

Reply:
Does not work when turned `ON`, so I agree with disabling.

Reply:
:+1:

Reply:
> This option does not work for me,

It seems to compile on my machine ... (and apparently on the jenkins node too) ....

Reply:
> It seems to compile on my machine...

Did you use fail on missing? It may have been disabled for you.

Reply:
```
mac-130277:master.debug pcanal$ grep -i bonjour CMakeCache.txt 
Bonjour_LIB_DEPENDS:STATIC=general;Core;
//Bonjour support, requires libdns_sd and/or Avahi
bonjour:BOOL=ON
mac-130277:master.debug pcanal$ ls -lart lib/libBonjour*
-rw-r--r--  1 pcanal  staff    2634 Oct  9 16:41 lib/libBonjour_rdict.pcm
-rw-r--r--  1 pcanal  staff     237 Oct  9 16:41 lib/libBonjour.rootmap
-rwxr-xr-x  1 pcanal  staff  153448 Oct  9 16:42 lib/libBonjour.so
```

Reply:
it also builds (for me) on High Sierra.

Reply:
Ok, thanks. I wonder if it works on the Linux nodes in Jenkins, or just the macs. I'll check later and post it here.